picard/mbxml.py

```diff
@@ -195,7 +195,7 @@ def artist_credit_to_metadata(node, m, release=False):
         m["musicbrainz_albumartistid"] = ids
         m["albumartist"] = artist
         m["albumartistsort"] = artistsort
-        m["~albumartists"] = artists
```

Comment:
I created ~albumartists.

I would suggest that we dispose of this now, and instead create update code that replaces `%_albumartists%` with `%albumartists%` .

It might break plugins, but with the exception of my `sort_multivalue_tags` plugin (which strangely already has `albumartists` instead of `~albumartists`) there are no other plugins I know of which use this.